index.js

```diff
@@ -1,52 +1,62 @@
-var path = require('path')
-var mkdirp = require('mkdirp')
-var includePathSearcher = require('include-path-searcher')
-var CachingWriter = require('broccoli-caching-writer')
-var sass = require('node-sass')
-var _ = require('lodash')
-var Promise = require('rsvp').Promise
-
-module.exports = SassCompiler
-SassCompiler.prototype = Object.create(CachingWriter.prototype)
-SassCompiler.prototype.constructor = SassCompiler
-function SassCompiler (inputTrees, inputFile, outputFile, options) {
-  if (!(this instanceof SassCompiler)) return new SassCompiler(inputTrees, inputFile, outputFile, options)
-  if (!Array.isArray(inputTrees)) throw new Error('Expected array for first argument - did you mean [tree] instead of tree?')
-
-  CachingWriter.call(this, inputTrees, options)
-
-  this.inputFile = inputFile
-  this.outputFile = outputFile
-  options = options || {}
-  this.sassOptions = {
-    imagePath: options.imagePath,
-    outputStyle: options.outputStyle,
-    sourceComments: options.sourceComments,
-    sourceMap: options.sourceMap,
-    precision: options.precision
-  }
-}
+var fs = require('fs');
+var path = require('path');
+var mkdirp = require('mkdirp');
+var includePathSearcher = require('include-path-searcher');
+var CachingWriter = require('broccoli-caching-writer');
+var sass = require('node-sass');
+var _ = require('lodash');
```

Comment:
although this is pre-existing, we should require the exact part of lodash that we need. Otherwise this can be a spin up hit.